zuul.d/infra-prod.yaml

```diff
@@ -324,3 +324,14 @@
       - inventory/
       - playbooks/install-helm-chart.yaml
       - playbooks/roles/install-helm-chart
+
+- job:
+    name: infra-prod-install-cce
+    parent: infra-prod-service-base
+    description: Install cloud CCE clusters
+    vars:
+      playbook_name: cloud-cce.yaml
+    files:
+      - inventory/
```

Comment:
let's have here really only inventory/service/group_vars/cloud-launcher.yaml not to call it every time we touch anything else in the inventory. This is very isolated from other inventory stuff that we can easily say what is important for us

Reply:
ok, done